fixture/client.json.proxy.json

```diff
@@ -3,62 +3,40 @@
   "target_body": "",
-  "proving": {
-    "manifest": {
-      "manifestVersion": "1",
-      "id": "reddit-user-karma",
-      "title": "Total Reddit Karma",
-      "description": "Generate a proof that you have a certain amount of karma",
-      "prepareUrl": "https://www.reddit.com/login/",
-      "request": {
-        "method": "GET",
-        "version": "HTTP/1.1",
-        "url": "https://gist.githubusercontent.com/mattes/23e64faadb5fd4b5112f379903d2572e/raw/74e517a60c21a5c11d94fec8b572f68addfade39/example.json",
-        "headers": {
-        },
-        "body": {
-          "userId": "<% userId %>"
-        },
-        "vars": {
-          "userId": {
-            "description": "Reddit username",
-            "required": true,
-            "pattern": "^[A-Za-z0-9_-]{3,20}$"
-          },
-          "authToken": {
-            "description": "Authentication token",
-            "required": false,
-            "default": "abcdef1234567890abcdef1234567890",
-            "pattern": "^[A-Za-z0-9]{32}$"
-          }
-        },
-        "extra": {
-          "headers": {
-            "User-Agent": "Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Mobile Safari/537.36",
-            "Content-Type": "application/json"
-          }
-        }
+  "target_method": "GET",
```

Comment:
i don't think we need these now, after the cleanup PR: #515

Reply:
These changes are not in anymore, I may have messed-up a rebase